### DISCUSSION TITLE
Caddy Page after install

### DISCUSSION THREAD

**Discussion by Brian21114** [mcguirepr89/BirdNET-Pi#1177] [General]
Just followed the instructions and after install I get a Caddy Works page. I'm using a pi 4 with Bullseye (Legacy, 64-bit) Lite.  Help please

**Comment by pbeskeen** [comment]
Same for me. Bookworm on Pi4B.

**Comment by pbeskeen** [comment]
I have subsequently tried the https://github.com/Nachtzuster/BirdNET-Pi fork and this installs on Bookworm and works successfully without this caddy issue being encountered.

**Reply by S-Tuthill** [reply]
Thanks for posting - this worked for me.

**Comment by pwagterberg** [comment]
The same for me with this caddy problem. Can someone help me with this?
Using raspberry 4b with 64 bit Lite version installed.

**Comment by Brian21114** [comment]
What I did was wipe the card and reinstall, did that 3 times before it booted right. I noticed if the install take 10min, it will fail. When it install right, it took about 20min.

**Reply by pwagterberg** [reply]
Thanks, it works.
